tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,10 +125,65 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_auth_email_checks(self):
+        # Setting invalid user.email
+        # newgitdir = mkdtemp()
+        # os.chdir(newgitdir)
+        # self.run_git_command('init')
+        self.run_git_command('config', 'user.email invalid#!format!*!@email.com')
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tests/vcs/git/GitCommitBearTest.py
+++ b/tests/vcs/git/GitCommitBearTest.py
@@ -130,7 +130,8 @@
         # newgitdir = mkdtemp()
         # os.chdir(newgitdir)
         # self.run_git_command('init')
-        self.run_git_command('config', 'user.email invalid#!format!*!@email.com')
+        self.run_git_command(
+            'config', 'user.email invalid#!format!*!@email.com')
 
         # Invalid email with no dns check
         # emailChange = run_shell_command('git config user.email')
```